src/main/include/io/clawIO.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
This file needs to be named `ClawIO.h` not `clawIO.h`. Capitalize the file names